pkgs/os-specific/linux/kernel/kernels-org.json

```diff
@@ -0,0 +1,38 @@
+{
+    "testing": {
+        "version": "6.6-rc1",
+        "hash": "02zh3dnikyhhlas9xccia963d4yqmzq0m4b8s10x8mjng3na45hd"
```

Comment:
Even though this is an internal data-structure, I don't like the naming here: `hash` expects either an SRI hash or something in the form of `<algo:base{32,64} repr>`, not a base32 sha256.

I'm fine with either solution, but please call this field `sha256` or fix the form of the value.

Reply:
I'm just hoping nix-prefetch-url starts producing SRI hashes eventually...